liquibase-core/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -1333,7 +1379,6 @@ protected void doMigration() throws Exception {
                 String result = snapshotCommand.execute().print();
                 outputWriter.write(result);
                 outputWriter.flush();
-                outputWriter.close();
```

Comment:
Why did you remove the outputWriter.close() in snapshot?

Reply:
I fat-fingered that one.  Not intentional.